draft-ietf-rats-uccs.md

```diff
@@ -475,6 +476,27 @@ an UCCS by enclosing it with a tag number TBD601:
 <!--  LocalWords:  Verifier's CWTs Attester Verifier FCFS
  -->
 
+# JSON Support
+
+The above definitions, concepts and security considerations all may be applied to define a JSON-encoded Claims-Set.
+Such an unsigned Claims-Set can be referred to as a "UJCS", an "Unprotected JWT Claims Set".
+The CDDL definition in {{fig-claims-set}} can be used for a "UJCS".
+
+~~~ cddl
+UJCS = Claims-Set
+~~~
+
+# EAT
```

Comment:
This probably needs references, expansion, etc.

Reply:
Add reference to EAT submods section. That's what you are after, right?

Reply:
The reference looks good to me ([this para](https://datatracker.ietf.org/doc/html/draft-ietf-rats-eat-22#section-4.2.18-15) of §4.2.18, specifically).

I don't think EAT needs to be expanded here, it's already mentioned in §4.1, §4.2 and §4.3.  More likely, when first introduced.

Reply:
LGTM

@cabo confirmed